docs/literate/src/files/first_steps/getting_started.jl

```diff
@@ -0,0 +1,240 @@
+
+
+# ### Modifying an existing setup
+
+# As an example, we will change the initial condition for calculations that occur in
+# `elixir_euler_ec.jl`. In this example we consider the compressible Euler equations in two spatial
+# dimensions,
+# ```math
+# \frac{\partial}{\partial t}
+# \begin{pmatrix}
+# \rho \\ \rho v_1 \\ \rho v_2 \\ \rho e
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial x}
+# \begin{pmatrix}
+# \rho v_1 \\ \rho v_1^2 + p \\ \rho v_1 v_2 \\ (\rho e +p) v_1
+# \end{pmatrix}
+# +
+# \frac{\partial}{\partial y}
+# \begin{pmatrix}
+# \rho v_2 \\ \rho v_1 v_2 \\ \rho v_2^2 + p \\ (\rho e +p) v_2
+# \end{pmatrix}
+# =
+# \begin{pmatrix}
+# 0 \\ 0 \\ 0 \\ 0
+# \end{pmatrix},
+# ```
+# for an ideal gas with the specific heat ratio ``\gamma``.
+# Here, ``\rho`` is the density, ``v_1`` and ``v_2`` are the velocities, ``e`` is the specific
+# total energy, and
+# ```math
+# p = (\gamma - 1) \left( \rho e - \frac{1}{2} \rho (v_1^2+v_2^2) \right)
+# ```
+# is the pressure.
+# Initial conditions consist of initial values for ``\rho``, ``\rho v_1``,
+# ``\rho v_2`` and ``\rho e``.
+# One of the common initial conditions for the compressible Euler equations is a simple density
+# wave. Let's implement it.
+
+# - Open the downloaded file `elixir_euler_ec.jl` with a text editor.
+# - Go to the line with the following code:
+#   ```julia
+#   initial_condition = initial_condition_weak_blast_wave
+#   ```
+#   Here, [`initial_condition_weak_blast_wave`](@ref) is used as the initial condition.
+# - Comment out the line using the `#` symbol:
+#   ```julia
+#   # initial_condition = initial_condition_weak_blast_wave
+#   ```
+# - Now you can create your own initial conditions. Add the following code after the
+#   commented line:
+
+    function initial_condition_density_waves(x, t, equations::CompressibleEulerEquations2D)
+        v1 = 0.1 # velocity along x-axis
+        v2 = 0.2 # velocity along y-axis
+        rho = 1.0 + 0.98 * sinpi(sum(x) - t * (v1 + v2)) # density wave profile
+        p = 20 # pressure
+        rho_e = p / (equations.gamma - 1) + 1/2 * rho * (v1^2 + v2^2)
+        return SVector(rho, rho*v1, rho*v2, rho_e)
+    end
+    initial_condition = initial_condition_density_waves
+
+# - Execute the following code one more time, but instead of `path/to/file` paste the path to the
+#   `elixir_euler_ec.jl` file that you just edited.
+#   ```julia
+#   using Trixi
+#   trixi_include(path/to/file)
+#   using Plots
+#   plot(sol)
+#   ```
+# Then you will obtain a new solution from running the simulation with a different initial
+# condition.
+
+trixi_include(@__MODULE__,joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_ec.jl"), #hide #md
+ initial_condition=initial_condition) #hide #md
+pd = PlotData2D(sol) #hide #md
+p1 = plot(pd["rho"]) #hide #md
+p2 = plot(pd["v1"], clim=(0.05, 0.15)) #hide #md
+p3 = plot(pd["v2"], clim=(0.15, 0.25)) #hide #md
+p4 = plot(pd["p"], clim=(10, 30)) #hide #md
+plot(p1, p2, p3, p4) #hide #md
```

Comment:
Does this give the same result as `plot(sol)`, which is suggested in the code above? If yes, why not use `plot(sol)`? If no, the user might wonder why the plot displayed in the tutorial is different than what she/he sees locally.

Reply:
Thank you very much for the review!

`plot(sol)` generates a noisy picture, because 3 of four terms remain approximately constant. Such noise can also confuse user...
![image](https://github.com/trixi-framework/Trixi.jl/assets/119304909/95d3db79-0854-4d2c-bcec-56c02ff766cf)

Reply:
Ok, then I suggest to also use this code also above such that the user gets the same non-noisy picture. See my suggestion.

Reply:
Target audience of tutorial is people with zero experience in Julia. So all called functions are explained, so that newbie could understand the purpose of every function call. And explaining `PlotData2D` and how to configure limits for every state variable seems to be to complicated in this case... May be leave `plot(sol)`, but add a comment, that 
```
pd = PlotData2D(sol)
p1 = plot(pd["rho"])
p2 = plot(pd["v1"], clim=(0.05, 0.15))
p3 = plot(pd["v2"], clim=(0.15, 0.25))
p4 = plot(pd["p"], clim=(10, 30))
plot(p1, p2, p3, p4)
```
should be execute to get exactly the same pictures?

Reply:
Yes, sounds good.